extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -815,8 +815,8 @@ private String deobfuscateSignature(String obfuscatedSig, String deobfuscationCo
         final Object result;
         try {
             final ScriptableObject scope = context.initSafeStandardObjects();
-            context.evaluateString(scope, deobfuscationCode, "decryptionCode", 1, null);
-            final Function deobfuscateFunc = (Function) scope.get("decrypt", scope);
+            context.evaluateString(scope, deobfuscationCode, "deobfuscationCode", 1, null);
```

Comment:
Same as above. Pretty sure you may not change this. However, you should consider extracting it into a global constant, too.

Reply:
`"deobfuscationCode"` is only the name of the JS file to be interpreted, which could be anything and for us has no effect whatsoever on code execution